bridge/client/app/_views/ktb-environment-view/ktb-stage-overview/ktb-services-list/ktb-services-list.component.html

```diff
@@ -83,7 +83,9 @@
           <ng-container *ngIf="row | toType: ServiceClass as service">
             <div *ngIf="service.latestSequence">
               <ktb-evaluation-info
-                [evaluation]="service.latestSequence.getEvaluationTrace(service.stage)"
+                *ngIf="{ value: service.latestSequence.getEvaluationTrace(service.stage) } as evaluation"
```

Comment:
This will always be true? I suppose this is only used to wrap the value in an object, which in turn can also be undefined or null?